cmd/govim/testdata/scenario_default/quickfix_replaced_dep.txt

```diff
@@ -0,0 +1,63 @@
+# Test that we get diagnostics for edits in a locally replaced dependency
+#
+# The scenario involves having a locally replaced dependency in go.mod.
+# When there is an error introduced into the local copy it should produce
+# a diagnostic (see https://golang.org/issues/36745).
+
+skip 'Pending fix for golang.org/issues/36745'
```

Comment:
This can now be changed to use the new condition:

```
[golang.org/issues/36745] skip
```

Reply:
Fixed